benches/benches/db_lookup_times_utils/utils.rs

```diff
@@ -0,0 +1,101 @@
+use fuel_core::{
+    database::{
+        database_description::on_chain::OnChain,
+        Database,
+    },
+    state::{
+        historical_rocksdb::StateRewindPolicy,
+        rocks_db::RocksDb,
+        IterableKeyValueView,
+    },
+};
+use fuel_core_storage::{
+    column::Column,
+    kv_store::{
+        KeyValueInspect,
+        StorageColumn,
+    },
+    tables::FullFuelBlocks,
+    StorageAsRef,
+};
+use fuel_core_types::{
+    blockchain::block::{
+        Block,
+        CompressedBlock,
+    },
+    fuel_tx::Transaction,
+    fuel_types::{
+        BlockHeight,
+        ChainId,
+    },
+};
+use rand::{
+    rngs::ThreadRng,
+    Rng,
+};
+use std::{
+    borrow::Cow,
+    path::Path,
+};
+
+pub fn thread_rng() -> ThreadRng {
+    rand::thread_rng()
+}
```

Comment:
What's the purpose of wrapping the `rand::thread_rng()` function here? This seems unnecessary to me.

Reply:
Also more generally, do we want to use the thread rng here? I tend to prefer to use seeded rngs for reproducibility in tests, but I'm not sure what's best practice for benchmarks.

Reply:
Right and you want to run the benchmarks multiple times with different randomness. Fair enough.

> I don't want to import rand::thread_rng into the db_lookup_times.rs benchmark file and hence wrapped over it.

If you want to avoid importing it, you can keep the namespace when calling it in the code like you do in this function. I often prefer that. Especially for a small crate like `rand`.

I don't see the benefit of wrapping the function compared to simply importing it. What is the problem with importing it?

Reply:
eh, just how I prefer to write code. will address.

Reply:
addressed in 61d866828

Reply:
> eh, just how I prefer to write code. will address.

Okay, interesting pattern. I'm not used to wrapping functions as an alternative to importing them. Personally I tend to keep the namespaces to a great extent, but this code base seems to prefer a lot of explicit imports. Thank you for accommodating this.